xml/System.Numerics/BitOperations.xml

```diff
@@ -671,9 +672,10 @@ This method is similar in behavior to the x86 instruction ROR.
         <Parameter Name="value" Type="System.UInt64" Index="0" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">The value.</param>
+        <summary>Round the given integral value up to a power of 2.</summary>
+        <returns>The smallest power of 2 which is greater than or equal to <paramref name="value" />.
+            If <paramref name="value" /> is 0 or the result overflows, returns 0.</returns>
```

Comment:
<param name="value">The value to round up to a power of two.</param>
        <summary>Rounds the specified <see langword="T:System.UInt64" /> value up to a power of two.</summary>
        <returns>The smallest power of two that's greater than or equal to <paramref name="value" />.

If <paramref name="value" /> is 0 or the result overflows, returns 0.</returns>